docs/en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -12,6 +12,7 @@ Our web-based [PHP installer](installation/) can check if you meet the requireme
  * Once PHP versions become [unsupported by the PHP Project](http://php.net/supported-versions.php),
    we drop support for those versions in the [next minor release](/contributing/release-process). This means that PHP 5.6 support may be dropped in a 4.x minor release after December 2018.
  * We recommend using a PHP accelerator or opcode cache, such as [xcache](http://xcache.lighttpd.net/) or [WinCache](http://www.iis.net/download/wincacheforphp).
+     * If you're using PHP >= 5.5 then [Zend OpCache](http://php.net/manual/en/book.opcache.php) is installed by default in managed package repos e.g. apt (You may need to manually enable it in php.ini). Do not try and run additional opcaches alongside Zend OpCache without first disabling it. 
```

Comment:
This definitely is not true for CentOS - I have to install it manually to get it in (`yum install php-opcache`)